Dockerfile

```diff
@@ -9,7 +9,6 @@ RUN cargo build --release
 RUN strip ./target/release/gtfs-geojson
 
 # https://github.com/etalab/transport-validator.git (rust app)
-FROM rust:latest as builder
```

Comment:
With the latest update of Docker (apparently, if I understand well), re-stating the same builder caused failures to find the cache.